src/components/CallView/CallView.vue

```diff
@@ -362,6 +362,13 @@ export default {
 		},
 	},
 	watch: {
+		'localCallParticipantModel.attributes.peerId'(newValue, previousValue) {
+			const index = this.screens.indexOf(previousValue)
+			if (index !== -1) {
+				this.screens.splice(index, 1, newValue)
```

Comment:
```suggestion
				this.screens[index] = newValue
```

Same result, but with direct approach will be better, i suppose

Reply:
:facepalm: Yes, totally agree, I do not know what I was thinking on xD